app/models/hotel_room.rb

```diff
@@ -0,0 +1,6 @@
+class HotelRoom < ApplicationRecord
+  belongs_to :hotel
+  belongs_to :room
+
+  has_many :reservation, dependent: :destroy
```

Comment:
Make sure you change this to `has_many :reservations`, having the table in singular may cause issues moving forward.